src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -13,7 +13,7 @@ case class RawlsUserSubjectId(value: String) extends UserAuthType
 case class RawlsGroupName(value: String) extends UserAuthType
 case class RawlsGroupEmail(value: String) extends UserAuthType
 case class RawlsBillingProjectName(value: String) extends UserAuthType
-case class RawlsGroupMemberList(userEmails: Seq[String], subGroupEmails: Seq[String])
+case class RawlsGroupMemberList(userEmails: Option[Seq[String]] = None, subGroupEmails: Option[Seq[String]] = None, userSubjectIds: Option[Seq[String]] = None, subGroupNames: Option[Seq[String]] = None)
```

Comment:
I disagree. The user knows what they are passing us so let them tell us and remove any ambiguity. It makes better sense to make these options in any case so the user does not need to specify sub groups if there aren't any.